### ISSUE TITLE
Accessibility issue: Elements must meet minimum color contrast ratio thresholds on /en/latest/examples/admonitions.html

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/sphinx_wagtail_theme#326]
## What
An accessibility scan flagged 3 elements on https://sphinx-wagtail-theme.readthedocs.io/en/latest/examples/admonitions.html because elements must meet minimum color contrast ratio thresholds. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/color-contrast?application=playwright.

The following elements need attention:

- `<p class="admonition-title">Important</p>` (selector: `.important > .admonition-title`)
- `<p class="admonition-title">See also</p>` (selector: `.seealso > .admonition-title`)
- `<p class="admonition-title">Tip</p>` (selector: `.tip > .admonition-title`)

To fix this, ensure the contrast between foreground and background colors meets wcag 2 aa minimum contrast ratio thresholds.

Specifically:

Fix any of the following:
-   Element has insufficient color contrast of 4.41 (foreground color: #ffffff, background color: #a456c9, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
